src/entities/user/lib/hooks/useFetchLogin.ts

```diff
@@ -0,0 +1,26 @@
+import { MutationOptions, useMutation } from "@tanstack/react-query"
+import { AxiosError, AxiosResponse } from "axios"
+
+import { BaseErrorResponse } from "~/utils/types/httpResponses"
+
+import { ILoginPayload } from "../../model/api.interfaces"
+import { Account, Authorization, User } from "../../model/interface"
+import { authorizationService } from "../authorization.service"
+
+export interface ILoginSuccessResponse {
+  account: Account
+  authToken: Authorization
+  message: string
+  user: User
+}
+
+export type SuccessLoginResponse = AxiosResponse<ILoginSuccessResponse>
+export type FailedLoginResponse = AxiosError<BaseErrorResponse>
+
+export const useFetchLogin = (options: MutationOptions<SuccessLoginResponse, FailedLoginResponse, ILoginPayload>) => {
```

Comment:
I think you should come up with a different naming strategy for hooks that are intended for POST/PUT operations.
"Fetch" implies that we get/receive something. But it doesn't imply that we change or create something.
You can go with names, for example, "useLoginMutation" (my favorite) or "useLogin" or something similiar

Reply:
Done